src/main/java/org/checkerframework/checker/androidresource/AndroidResourceAnnotatedTypeFactory.java

```diff
@@ -259,11 +267,56 @@
     protected final AnnotationMirror XML_CONTAINER =
             AnnotationBuilder.fromClass(elements, XmlContainer.class);
 
+    /**
+     * Contains Android package prefixes for the actual annotations.
+     */
+    private static final List<String> ANDROID_PACKAGE_PREFIXES = Arrays.asList(
+            // https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/annotation/
+            "android.annotation.",
+            // https://developer.android.com/reference/android/support/annotation/package-summary
+            "android.support.annotation.",
+            // https://developer.android.com/reference/androidx/annotation/package-summary
+            "androidx.annotation.");
+
     public AndroidResourceAnnotatedTypeFactory(BaseTypeChecker checker) {
         super(checker);
+        makeAndroidAnnotationsAliasesToChecker();
         postInit();
     }
 
+    /**
+     * Make the Android <code>@XXXRes</code> annotations aliases to <i>android-resource-checker</i> annotations.
+     */
+    private void makeAndroidAnnotationsAliasesToChecker() {
+
+        ANDROID_PACKAGE_PREFIXES.forEach((annotation) -> {
+            addAliasedAnnotation(annotation + "AnyRes", ANY_RES);

```

Comment:
I wonder if you can use another lambda to further reduce the duplication in this code - map over (String, Annotation) and map to the expression here.

Reply:
Fixed.